lambda/templates/repo-config/package.json.hbs

```diff
@@ -1,23 +1,38 @@
 {
   "name": "{{ kebabCase criName }}",
   "description": "",
+  "author": "GOV.UK One Login",
+  "workspaces": [
+    "lambdas/*"
+  ],
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "unit": "jest --silent",
     "test": "npm run unit",
     "test:coverage": "npm run unit -- --coverage",
     "pre-commit": "pre-commit run --all-files --show-diff-on-failure",
-    "pc": "npm run pre-commit"
+    "pc": "npm run pre-commit",
```

Comment:
There's no dependency for `pre-commit` and we shouldn't need to run this normally in the generated repo.

(I admit, the workflow when running tempaltes + validating correctness is weird, and required a lot of `pre-commit run --all-files --show-diff-on-failure`)

Reply:
It is useful to be able to run this at any time - not just when you commit, and it's good to encourage people to use the tools we have configured.
Now, each dev can make this is an alias on their machine so it's not a big deal if it's not included but I just think it's useful to make it easy for people to run checks on their local branches.

Reply:
If people want to run pre-commit manually at any time, they can do it. We don't need to pollute the package.json with extra runners. If pre-commit changes their API in version 4, we'll have to update every single repo for this shortcut.

As you said, if developers find this useful - and don't want to reuse their history - they can create an alias on their machine.